Automattic/jetpack
# AI Assistant: Fix try again behavior on inline extensions

Fixes #37249

## Proposed changes:

* Saves the last request to use it on retry

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor and add a Heading block with some content
* Disable the network on the browser's tools
* Send a manual request and wait for it to fail
* Click on "try again"
* Check that the same request was sent
* Try it on the toolbar actions
* For the toolbar actions, check that the placeholder changes during request and is reset on failure or success
